src/js/player.js

```diff
@@ -423,6 +425,24 @@ class Player extends Component {
    * @method updateStyleEl_
    */
   updateStyleEl_() {
+    if (window.VIDEOJS_NO_DYNAMIC_STYLE === true) {
+      /* jshint ignore:start */
+      let {width_: width = this.options_.width, height_: height = this.options_.width} = this;
```

Comment:
For those to read this in the future, what's happening is that it sets two variables called `width` and `height` to be the values of `this.width_` and `this.height_`, they also have default values of `this.options_.width` and `.this.options_.height` if `this.width_` and `this.height_` aren't set or are falsey.
That last bit is actually why I'm going to change it because we need to account for the possibility that the user wanted to set the width to be `0`. Why? Not sure, but it should still work.